core/src/env/traits.rs

```diff
@@ -48,8 +53,13 @@ pub trait EnvTypes {
     type Moment: Codec + Clone + PartialEq + Eq + core::fmt::Debug;
     /// The type of block number.
     type BlockNumber: Codec + Clone + PartialEq + Eq + core::fmt::Debug;
+    /// The type of a call into the runtime
+    type Call: Codec + Clone + PartialEq + Eq + core::fmt::Debug;
 }
 
+/// Substrate runtime dispatchable call
+pub trait Call<T: EnvTypes>: Encode {}
```

Comment:
What is this used for? Please add this rational to dev-docs.

Reply:
Removed